pkg/lib/verifier/rule_factory.go

```diff
@@ -83,50 +83,44 @@ func (factory *ruleFactory) GetVerificationRules() ([]rules.Rule, string, error)
 
 	}
```

Comment:
This code needs to be refactored.  Originally, all of the rule building was delegated to the 'ruleBuilder' interface (ex. GetPlatformRules).  Now, these functions are mostly empty in ESXI and in some cases return "builtin" rules like AIK verification.